queue_services/entity-emailer/src/entity_emailer/message_tracker/tracker.py

```diff
@@ -97,6 +97,14 @@ def get_message_context_properties(queue_msg: nats.aio.client.Msg):
             message_id = f'{etype}_{option}_{filing_id}'
             return create_message_context_properties(etype, message_id, None, None, False)
 
+        if etype == 'agmExtension' \
+                and (option := email.get('option', None)) \
+                and option == 'COMPLETED' \
+                and (filing_id := email.get('filingId', None)):
+            # option contains current status of filing - COMPLETED
+            message_id = f'{etype}_{option}_{filing_id}'
+            return create_message_context_properties(etype, message_id, None, None, False)
```

Comment:
I think we can combine this code with `agmLocationChange` logic above.

Maybe just use something like `if etype in ('agmLocationChange', 'agmExtension')...`